modules/sno-du-applying-the-extra-installation-manifests.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/sno-du-deploying-clusters-on-single-nodes.adoc
+
+:_content-type: CONCEPT
+[id="sno-du-applying-the-extra-installation-manifests_{context}"]
+= Applying the extra installation manifests
+
+To apply the distributed unit (DU) configuration to the single node cluster, the following
+extra installation manifests need to be included during installation:
+
+* Enable workload partitioning.
+* Other MachineConfigs – It is recommended, but not required, to apply these custom resources (CRs)
```

Comment:
Which other machine configs? 
s/MachineConfigs/machine configs or `MachineConfig` objects
Why would I not apply them?

Reply:
Other machineconfigs are located in the provided repo.

Reply:
@yliu127 @imiller0 Please confirm.

Reply:
There are a set of MachineConfig CRs that we include by default. The user may choose to include additional MachineConfig CRs that are unique to their environment. If they choose to include their own MachineConfigs then we recommend they include them during installation.

Reply:
Added Ian's comment.